Deprecate workspace_external_id/workspace_external_ids in tfe_notification_configuration, tfe_policy_set, and tfe_run_trigger resources (#182)
terraform-providers/terraform-provider-tfe: CHANGELOG.md

## 0.18.0 (Unreleased)

DEPRECATIONS:

> * d/tfe_workspace_ids: Added deprecation warning to `ids` attribute, preferring `full_names` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))

Reviewer: ```suggestion
* d/tfe_workspace_ids: Added deprecation warning to the `ids` attribute, preferring `full_names` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))
```

> * r/tfe_notification_configuration: Added deprecation warning to `workspace_external_id` attribute, preferring `workspace_id` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))

Reviewer: ```suggestion
* r/tfe_notification_configuration: Added deprecation warning to the `workspace_external_id` attribute, preferring `workspace_id` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))
```

> * r/tfe_policy_set: Added deprecation warning to `workspace_external_ids` attribute, preferring `workspace_ids` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))

Reviewer: ```suggestion
* r/tfe_policy_set: Added deprecation warning to the `workspace_external_ids` attribute, preferring `workspace_ids` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))
```

> * r/tfe_run_trigger: Added deprecation warning to `workspace_external_id` attribute, preferring `workspace_id` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))

Reviewer: ```suggestion
* r/tfe_run_trigger: Added deprecation warning to the `workspace_external_id` attribute, preferring `workspace_id` instead ([#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182))
```

NOTES:
* More information about these deprecations can be found in the description of [#182](https://github.com/terraform-providers/terraform-provider-tfe/pull/182)

> * d/tfe_workspace_ids: The deprecation warning for the ids attribute will not go away until it is removed. 

Reviewer: ```suggestion
* d/tfe_workspace_ids: The deprecation warning for the `ids` attribute will not go away until it is removed. 
```

This is due to a [limitation of the 1.0 version of the Terraform SDK](https://github.com/hashicorp/terraform/issues/7569) for deprecation warnings on attributes that aren't specified in a configuration.

> If you have made sure to change all references to this data source's `ids` attribute to the new `full_names` attribute, you can ignore the warning.  

Reviewer: ```suggestion
If you have already changed all references to this data source's `ids` attribute to the new `full_names` attribute, you can ignore the warning.  
```
